### PR TITLE
Add time sync example

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

**Comment by Acuadros95** [comment]
@pablogs9 I think we should update the micro_ros_arduino lib release ([link](https://github.com/micro-ROS/micro_ros_arduino/releases))

File: examples/micro-ros_time_sync/micro-ros_time_sync.ino [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  //create init_options
+  RCCHECK(rclc_support_init(&support, 0, NULL, &allocator));
+
+  // Synchronize time
+  RCCHECK(rmw_uros_sync_session(timeout_ms));
+}
+
+void loop() {
+  time_ms = rmw_uros_epoch_millis(); 
+
```
Comment (pablogs9) [inline examples/micro-ros_time_sync/micro-ros_time_sync.ino:51]:
```suggestion
      RCCHECK(rmw_uros_sync_session(timeout_ms));
```

File: examples/micro-ros_time_sync/micro-ros_time_sync.ino [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  if (time_ms <= 0)
+  {
+      // Error on time sync, retrying.
+      RCCHECK(rmw_uros_sync_session(timeout_ms));
+  }
```
Comment (pablogs9) [inline examples/micro-ros_time_sync/micro-ros_time_sync.ino:56]:
```suggestion
```

File: examples/micro-ros_time_sync/micro-ros_time_sync.ino [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  //create init_options
+  RCCHECK(rclc_support_init(&support, 0, NULL, &allocator));
+
+  // Synchronize time
+  RCCHECK(rmw_uros_sync_session(timeout_ms));
+}
+
+void loop() {
+  time_ms = rmw_uros_epoch_millis(); 
+      RCCHECK(rmw_uros_sync_session(timeout_ms));
```
Comment (pablogs9) [inline examples/micro-ros_time_sync/micro-ros_time_sync.ino:51]: Indentation

File: examples/micro-ros_time_sync/micro-ros_time_sync.ino [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  else
```
Comment (pablogs9) [inline examples/micro-ros_time_sync/micro-ros_time_sync.ino:52]: This else is isolated

**Review by pablogs9 — CHANGES_REQUESTED** [review]

**Comment by pablogs9** [comment]
Test are failing

**Pull request merged**